shared/src/packages/sections/libs/validation-schemas/site-footer-update-content.validation-schema.ts

```diff
@@ -0,0 +1,21 @@
+import joi from 'joi';
+
+import { SectionValidationMessage } from '~/packages/sections/libs/enums/enums.js';
+
+import { type SiteFooterUpdateContentDto } from '../types/types.js';
+
+const siteFooterUpdateContent = joi.object<SiteFooterUpdateContentDto, true>({
+  contacts: joi.object({
```

Comment:
you should also add other properties (logo, description) and add this functionality on frontend